CryptoPkg/Library/Include/CrtLibSupport.h

```diff
@@ -437,6 +472,66 @@ strcat (
   const char  *strSource
   );
 
+int
+fflush (
+  FILE  *stream
+  );
+
+int
+ferror (
+  FILE  *fp
+  );
+
+int
+fseek (
+  FILE  *stream,
+  long  offset,
+  int   whence
+  );
+
+int
+feof (
+  FILE  *fp
+  );
+
+char *
+fgets (
+  char  *str,
+  int   n,
+  FILE  *stream
+  );
+
+int
+ftell (
+  FILE  *fp
+  );
+
+int
+fputs (
+  const char  *str,
+  FILE        *stream
+  );
+
+char *
+strdup (
+  char  *strSource

```

Comment:
```suggestion
  char  *s
```
strdup(3) name choice.

A shame, but it is what it is.